samples/Foundatio.ReceiverConsole/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Foundatio.Messaging;
+using Foundatio.Queues;
+using System.Threading.Tasks;
+
+namespace Foundatio.ReceiverConsole {
+    class Program {
+        static void Main(string[] args) {
+           
+
+            try {
+                var o = new Receiver();
```

Comment:
"o"? 🙂

Reply:
We can now do static `async Task MainAsync` and remove the Getawaiter().GetResult(). As this was added in the latest .net core..

Reply:
You can also `async Task Main()` 🙂

Reply:
Good tip Blake. I just realized i have removed the reference of sample project from the solution but didnt delete the sample files from the repo. Will be deleting all the samples. I think our unit test cases are great enough to test all the possible scenarios